src/sql/schema169.sql

```diff
@@ -49,6 +49,12 @@ CREATE TABLE IF NOT EXISTS compounds (
   `is_cmr` TINYINT UNSIGNED NOT NULL DEFAULT 0,
   `is_nano` TINYINT UNSIGNED NOT NULL DEFAULT 0,
   `is_controlled` TINYINT UNSIGNED NOT NULL DEFAULT 0,
+  `is_ed2health` TINYINT UNSIGNED NOT NULL DEFAULT 0,
```

Comment:
you cannot edit an already published schema. Please use `bin/console dev:genschema` (and chown the files with your user) to generate a new schema that will apply to existing instances